Source/WebKit/UIProcess/API/Cocoa/NSAttributedString.mm

```diff
@@ -133,12 +137,28 @@ @implementation _WKAttributedStringWebViewCache
     return configuration;
 }
 
++ (RetainPtr<NSMutableArray<NSURL *>>)readOnlyAccessPaths
```

Comment:
I think should just return a pointer, not a `RetainPtr`. It’s not transferring ownership.

I think this should just be a function, not a method.

Reply:
Okay, I'll switch back to pointer and make this a static function.